addons/mail/static/src/owl/components/thread_preview/thread_preview.js

```diff
@@ -4,9 +4,10 @@ odoo.define('mail.component.ThreadPreview', function (require) {
 const MessageAuthorPrefix = require('mail.component.MessageAuthorPrefix');
 const PartnerImStatusIcon = require('mail.component.PartnerImStatusIcon');
 const mailUtils = require('mail.utils');
+const { useStoreCompareKeys } = require('mail.hooks.useStoreCompareKeys');
```

Comment:
`mail.hooks` before `mail.utils`

Reply:
What is the guideline?

Reply:
Ordered by the name inside the require apparently.